### DISCUSSION TITLE
lint error

### DISCUSSION THREAD

**Discussion by jang-jaehyuk** [simple-icons/simple-icons#6919] [Help with SVGs]
This is the result of npm run lint

x icons/thealgorithms.svg
  x extraneous Unexpected character(s), most likely extraneous whitespace, dete
               cted in SVG markup

my svgcode is `<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>The Algorithms</title><path d="M8.226 19.857H1.353l1.79-4.225h4.812L13.308 3.21H7.564l-4.226 9.82h2.587c.18.3.511.511.887.511a1.04 1.04 0 0 0 1.038-1.037c0-.572-.467-1.023-1.038-1.023-.421 0-.767.24-.932.602H4.647l3.503-7.94h3.76L7.384 14.684l-4.767.03L0 20.79h8.842L10 18.263h3.835l1.278 2.526H24L15.985 3.211Zm2.27-2.586 1.384-3.023 1.503 3.023zm5.218 2.691-.872-1.759h2.737c.18.33.526.556.917.556a1.04 1.04 0 0 0 1.038-1.037 1.04 1.04 0 0 0-1.038-1.038c-.42 0-.782.256-.947.617H14.42l-2.09-4.06 1.534-3.369 1.729 3.52h.812c.165.345.526.6.932.6a1.04 1.04 0 0 0 1.038-1.037 1.04 1.04 0 0 0-1.038-1.037 1.03 1.03 0 0 0-.962.661h-.3l-1.79-3.639 1.7-3.73 6.676 14.752z"/></svg>`

I can't find what's wrong with this error.
Please help me..

**Comment by killshot13** [comment]
Try this version, it should work now. All I did is take your code and run it through [svgo](https://github.com/svg/svgo) to minify it. 

```xml
<svg viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><path d="M8.226 19.857H1.
  353l1.79-4.225h4.812L13.308 3.21H7.564l-4.226 9.82h2.587c.18.3.511.511.887.511a1.
  04 1.04 0 0 0 1.038-1.037c0-.572-.467-1.023-1.038-1.023-.421 0-.767.24-.932.602H4.
  647l3.503-7.94h3.76L7.384 14.684l-4.767.03L0 20.79h8.842L10 18.263h3.835l1.278 2.
  526H24L15.985 3.211Zm2.27-2.586 1.384-3.023 1.503 3.023zm5.218 2.691-.8721.759h2.
  737c.18.33.526.556.917.556a1.04 1.04 0 0 0 1.038-1.037 1.04 1.04 0 0 0-1.038-1.
  038c-.42 0-.782.256-.947.617H14.42l-2.09-4.06 1.534-3.369 1.729 3.52h.812c.165.345.
  526.6.932.6a1.04 1.04 0 0 0 1.038-1.037 1.04 1.04 0 0 0-1.038-1.037 1.03 1.03 0 0 
  0-.962.661h-.3l-1.79-3.639 1.7-3.73 6.676 14.752z"/></svg>
```

Regardless, the issue is probably either the `role` or the `title` attribute in your SVG code. Or both. Props to you for trying to support accessibility, but not every piece of software out there is going to be up to be speed or even agree on the best way of doing so.

Ref: [Contextually Marking up accessible images and SVGs](https://www.scottohara.me/blog/2019/05/22/contextual-images-svgs-and-a11y.html)

**Reply by sachinraja** [reply]
Thanks for helping! Looks like this was already solved in https://github.com/simple-icons/simple-icons/pull/6917#discussion_r757772654. Also we actually require `role` and `title`.
